Library/Formula/supercollider.rb

```diff
@@ -0,0 +1,61 @@
+
+class Supercollider < Formula
+  homepage "https://supercollider.github.io"
+  url "https://github.com/supercollider/supercollider.git",
+    :tag => "after-qt5",
+    :revision => "085cab288947fe06f468117f5c294db516b2afc0"
+  sha256 "80956465c2637cf0cc67c3550900e2f392d10d33f2518c25bbb9332901ae226c"
```

Comment:
actually when I tried to remove revision it complained and said that both tag and revision are required.